bundles/org.openhab.binding.meater/src/main/java/org/openhab/binding/meater/internal/MeaterException.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.meater.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * {@link MeaterException} is used when there is exception communicating with Meater REST API.
+ *
+ * @author Jan Gustafsson - Initial contribution
+ */
+@NonNullByDefault
+public class MeaterException extends Exception {
+
+    private static final long serialVersionUID = 2543564118231301158L;
+
+    public MeaterException(Exception source) {
+        super(source);
+    }
+
+    public MeaterException(String message) {
+        super(message);
+    }
```

Comment:
Consider also adding this overload:

```java
    public MeaterException(String message, Throwable cause) {
        super(message, cause);
    }
```

this would add opportunity to pass original exception.

Reply:
Added.